pkg/orchestrator/etcd_worker.go

```diff
@@ -45,6 +47,8 @@ type EtcdWorker struct {
 	barrierRev int64
 	// prefix is the scope of Etcd watch
 	prefix util.EtcdPrefix
+
+	deleteCounter int64
```

Comment:
can you add a comment about `deleteCounter`, it maybe hard for other to get the reason why we need it.